lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -50,7 +50,17 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
-  <button class="button button-primary button-block"
+  <div class="modal-footer-warning"
+    ng-show="modal.state.hasDrive() && modal.state.isSystemDrive(modal.state.getDrive())">
```

Comment:
If we're having a `.showFooterWarning()`, should we also have a `.getFooterWarning()`, instead of embedding `Etcher has identified this as a system drive` in this file?
Could then extend the model to show other warnings, without having to edit this file.

Or we could have a generic `.getFooterMessage()` (ties into your earlier "I imagine this could be used for something else in the future" comment), and a function to get the 'severity' of the FooterMessage? (which would determine if the button should be red or not)

Or perhaps taking things one step further, the model could have a `.getButtonStatus()` function (independent of `.getFooterMessage()`), which would determine whether the button is disabled (grey), enabled (blue) or warning (red).